README.md

```diff
@@ -92,6 +92,60 @@ module "lambda_with_specific_provider" {
 
+#### RDS postgres database
+```hcl
+module "rds_postgres_database" {
+  # source = "github.com/THEY-Consulting/they-terraform//azure/database/rds"
+  source = "../../../aws/database/rds"
+
+  db_name        = "postgres" #DBName must begin with a letter and contain only alphanumeric characters
+  engine         = "postgres"
+  engine_version = "15.5"
+  user_name      = "psql"
+  password       = sensitive("Passw0rd123!")
+
+  allocated_storage = 5
+  instance_class    = "db.t3.micro"
+
+  publicly_accessible = true
+
+
+  tags = {
+    Project   = "they-terraform-examples"
+    CreatedBy = "terraform"
+  }
+}
+```
+#### Inputs
+
+| Variable                      | Type         | Description                                                                        | Required | Default                    |
+|-------------------------------|--------------|------------------------------------------------------------------------------------|----------|----------------------------|
+| db_name                       | string       | Name of the database                                                               | yes      |                            |
+| engine                        | string       | Engine of the database                                                             | yes      | `"postgres"`               |
+| engine_version                | string       | Database's engine version                                                          | yes      | `"15.5"`                   |
+| user_name                     | string       | Main username for the database                                                     | yes      |                            |
+| password                      | string       | Password of the main username for the database                                     | yes      |                            |
+| allocated_storage             | number       | Allocated storage for the DB in GBs.                                               | yes      | `5`                        |
+| max_allocated_storage         | number       | Upper limit to which the RDS can automatically scale the storage of the db instance| no       | `30`                       |
+| instance_class                | string       | Instacnce class of database                                                        | yes      | `"db.t3.micro"`            |
+| multiple_az                   | bool         | Specifies whether the RDS is multi-AZ                                              | yes      | `false`                    |
+| storage_type                  | string       | Database's storage type                                                            | yes      | `"gp2"`                    |
+| backup_retention_period       | number       | The number of days to retain backups for                                           | no       | `1`                        |
```

Comment:
yeah I was thinking to use 1, because for now we're only thinking of using this for dev, so I am not imagining databases that are constantly in use, therefore I thought 1 day as  a default was quite ok, but I am not at all attached to that, so we can increase the default to 14.